examples/test_resource_controller_v2_examples.py

```diff
@@ -175,11 +176,22 @@ def test_list_resource_instances_example(self):
             print('\nlist_resource_instances() result:')
             # begin-list_resource_instances
 
-            resource_instances_list = resource_controller_service.list_resource_instances(
-                name=resource_instance_name
-            ).get_result()
+            next_page_id = None
+
+            while True:
+                resource_instances_list = resource_controller_service.list_resource_instances(
+                    name=resource_instance_name,
+                    limit=100,
+                    start=next_page_id,
+                ).get_result()
+
+                print(json.dumps(resource_instances_list, indent=2))
+
+                next_page_url = resource_instances_list.get('next_url')
+                if next_page_url is None:
+                    break
 
```

Comment:
```suggestion
```
I think it would be better if we did the if-check to exit the loop on the next_page_id value that we get back from get_query_param(), instead of on the next_page_url value.  What do you think?